app/src/main/java/org/mozilla/focus/browser/VideoVoiceCommandMediaSession.kt

```diff
@@ -320,11 +327,33 @@ var _firefoxTV_isPlaybackStateObserverLoaded;
     }
 
     function syncPlaybackState() {
-        _firefoxTV_playbackStateObserverJava.setIsAnyVideoPlaying(isAnyVideoPlaying());
+        let isVideoPresent;
+        let isPlaying;
+        let positionSeconds;
+        let playbackRate; /* 0.5, 1, etc. */
+        const maybeTargetVideo = getMaybeTargetVideo();
+        if (maybeTargetVideo) {
+            isVideoPresent = true;
+            isPlaying = !maybeTargetVideo.paused;
+            positionSeconds = maybeTargetVideo.currentTime;
+            playbackRate = maybeTargetVideo.playbackRate;
+        } else {
+            isVideoPresent = false;
+            isPlaying = false;
+            positionSeconds = null;
+            playbackRate = null;
+        }
+
+        _firefoxTV_playbackStateObserverJava.syncPlaybackState(
+            isVideoPresent, isPlaying, positionSeconds, playbackRate);
     }
 
-    function isAnyVideoPlaying() {
-        return Array.from(document.querySelectorAll('video')).some(video => !video.paused);
+    function getMaybeTargetVideo() {
```

Comment:
Why "MaybeTarget"? How about getActiveVideo? or getMaybeActiveVideo?

Reply:
I'm picking the video based on a heuristic: I think I used "target" unconsciously as way of indicating this because it's hard to quantify what the heuristic is briefly. It's not an "active" video because if there is no playing video, we get the first one in the page.

Maybe, `getMaybePlayingVideoOrFirstInPage()`? :)